packages/website/docs/Future-Features.md

```diff
@@ -79,3 +79,32 @@ Right now, Monte Carlo simulations are totally random. It would be nicer to be a
 
 ## Fixes
 - Discrete distributions are particularly buggy. Try ``mm(1,2,3,4,5,6,7,8,9,10) .* (5 to 8)``
+
+## New Functions
+
+### Distributions
+```
+cauchy() //todo
+pareto() //todo
+metalog() //todo
+```
+
+Possibly change mm to mix, or mx(). Also, change input format, maybe to mx([a,b,c], [a,b,c]).
+
+
+### Functions
+```
```

Comment:
some of our docs use `js` in codeblock because it's close enough for many subsets of squiggle.

Reply:
`python` also might work? it's mainly nice for syntax highlighting function names distinct from parens and arguments.